skopt/optimizer/optimizer.py

```diff
@@ -44,10 +48,13 @@ class Optimizer(object):
         - an instance of a `Dimension` object (`Real`, `Integer` or
           `Categorical`).
 
-    * `base_estimator` [sklearn regressor]:
+    * `base_estimator` ["GP", "RF", "ET", "GBRT" or sklearn regressor, default="GP"]:
         Should inherit from `sklearn.base.RegressorMixin`.
         In addition the `predict` method, should have an optional `return_std`
         argument, which returns `std(Y | x)`` along with `E[Y | x]`.
+        If provided base_estimator is in ["GP", "RF", "ET", "GBRT"]
+        then the corresponding estimator in the minimize functions is used
+        as a surrogate model.
```

Comment:
-> "If base_estimator is one of [GP, ET,...] a default surrogate model of the corresponding type is used. The defaults correspond to what is used in the `*_minimize` functions.

Still not perfect, but I think already an improvement. Need to think a bit more about wording.